utils/dbutils.py

```diff
@@ -77,7 +77,7 @@ def getGEMDBView(view, vfatList=None, debug=False):
 
     if vfatList is not None:
         # First warn the user which VFATs are *not* found
-        if len(vfatList) != dfGEMView.shape[1]:
+        if len(vfatList) != dfGEMView.shape[0]:
```

Comment:
for my curiosity, what is the dimensionality of `dfGEMView`?

Reply:
The first index refers to the rows and the second one to the columns. In our case, the dimensionality is therefore (number of VFATs retrieved from DB, number of columns in the requested view).